src/Mealz/AccountingBundle/Tests/Controller/CostSheetControllerTest.php

```diff
@@ -93,4 +93,40 @@ public function testHashRemoveFromDatabase(): void
         $this->assertEquals(0, $balanceAfter);
         $this->assertNull($profile->getSettlementHash());
     }
+
+    public function testHideUserRequestWithNonHiddenUser(): void {
+        // Pre-action tests
+        $profile = $this->getUserProfile(parent::USER_STANDARD);
+        $this->assertNotNull($profile);
+        $this->assertFalse($profile->isHidden());
+
+        // Trigger action
+        $this->client->request('GET', '/print/costsheet/hideuser/request/' . $profile->getUsername());
+        $this->assertTrue($this->client->getResponse()->isSuccessful());
+
+        // Check after action
+        $profile = $this->getUserProfile(parent::USER_STANDARD);
+        $this->assertTrue($profile->isHidden());
+    }
+
+    public function testHideUserRequestWithHiddenUser(): void {
+        // Pre-action tests
+        $profile = $this->getUserProfile(parent::USER_STANDARD);
+        $this->assertNotNull($profile);
```

Comment:
Do we need this assertion? The "getUserProfile()" method always returns a Profile, if it exists. Else it throws "user not found" exception.